src/components/fields/password-field/password-field.js

```diff
@@ -72,16 +80,39 @@ class PasswordField extends React.Component {
     return (
       <Constraints.Horizontal constraint={this.props.horizontalConstraint}>
         <Spacings.Stack scale="xs">
-          <FieldLabel
-            title={this.props.title}
-            hint={this.props.hint}
-            description={this.props.description}
-            onInfoButtonClick={this.props.onInfoButtonClick}
-            hintIcon={this.props.hintIcon}
-            badge={this.props.badge}
-            hasRequiredIndicator={this.props.isRequired}
-            htmlFor={this.state.id}
-          />
+          <Spacings.Inline
+            alignItems="center"
+            css={css`
+              justify-content: space-between;
```

Comment:
spacings don't accept a className prop

[ref](https://github.com/commercetools/ui-kit/blob/master/src/components/spacings/inline/inline.js)

Reply:
After #557 is merged, you can simply pass `justifyContent` as a prop

Reply:
Just waiting for it to be merged and I'll update this.

Reply:
@jonnybel alright, PR is merged. You can rebase and use the new prop here 🚀